sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -108,6 +118,136 @@ public ContactDirectorySteps(
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
         });
 
+    When(
+        "I click on New Entry in Exposure Details Known",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(NEW_ENTRY_EPIDEMIOLOGICAL_DATA);
+          TimeUnit.SECONDS.sleep(4);
+        });
+    When(
+        "I click on edit Exposure vision button",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.clickOnWebElementBySelector(OPEN_SAVED_EXPOSURE_BUTTON);
+          TimeUnit.SECONDS.sleep(2);
+        });
+    When(
+        "I click on the Epidemiological Data navbar field",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EPIDEMIOLOGICAL_DATA_TAB);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
+        });
+    When(
+        "I click on Indoors with {string} option in Exposure Details",
+        (String indoorsOption) -> {
+          webDriverHelpers.clickWebElementByText(INDOORS_OPTION, indoorsOption);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
+        });
+
+    When(
+        "I select all options in Type of activity from Combobox in Exposure form",
+        () -> {
+          loopByActivityTypes();
+          TimeUnit.SECONDS.sleep(2);
+        });
+
+    When(
+        "I select ([^\"]*) option in Type of activity from Combobox in Exposure form",
+        (String typeOfactivity) -> {
+          webDriverHelpers.selectFromCombobox(TYPE_OF_ACTIVITY_COMBOBOX, typeOfactivity);
+          TimeUnit.SECONDS.sleep(2);
+        });
+
+    When(
+        "I check all Type of gathering from Combobox in Exposure form",
+        () -> {
+          loopByGatheringType();
```

Comment:
let's use select instead of check to avoid confusions between an assert action and a click action (I had this problem with other PRs but now I realised it, sorry)